Makefile

```diff
@@ -15,8 +15,10 @@ BROWSER := python -c "$$BROWSER_PYSCRIPT"
 VENV_PATH = env
 VENV = $(VENV_PATH)/ready
 BIN = $(VENV_PATH)/bin
+VENV_BIN = $(BIN)
```

Comment:
Given we never use `VENV_BIN`, it just happens to be the env var expected in a subshell, can we instead assign it to `BIN` within the export?

e.g.
```make
export VENV_BIN=$(BIN)
```